pkg/rootless/rootless_linux.go

```diff
@@ -237,46 +220,15 @@ func BecomeRootInUserNS() (bool, int, error) {
 		return false, -1, errors.Errorf("cannot re-exec process")
 	}
 
-	allowSingleIDMapping := os.Getenv("PODMAN_ALLOW_SINGLE_ID_MAPPING_IN_USERNS") != ""
-
 	var uids, gids []idtools.IDMap
 	username := os.Getenv("USER")
 	if username == "" {
 		user, err := user.LookupId(fmt.Sprintf("%d", os.Getuid()))
```

Comment:
I'm going back and forth so I'll throw it up for consideration.  Any value to doing a logru.debugf here and after line 231 to show the error that was hit?

Reply:
that means the UID we are running with is not present in the users DB.  I don't think it should matter, but I can add a Debugf if that can help